auth/src/main/AndroidManifest.xml

```diff
@@ -80,5 +80,9 @@
             </intent-filter>
         </activity>
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
```

Comment:
Is this actually necessary?  In the Firebase Auth quickstart I was able to get Twitter Sign In to work with only the consumer key/secret and didn't need the app ID or any meta-data:

https://github.com/firebase/quickstart-android/blob/master/auth/app/src/main/AndroidManifest.xml